recipes/liblifthttp/all/test_package/conanfile.py

```diff
@@ -9,12 +9,16 @@ class TestPackageConan(ConanFile):
     generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
     test_type = "explicit"
 
-    def requirements(self):
-        self.requires(self.tested_reference_str)
-
     def layout(self):
         cmake_layout(self)
 
+    def configure(self):
+        if self.settings.os == "Windows":
+            self.options["libcurl/*"].with_ssl = "schannel"
+
```

Comment:
```suggestion
```

Let's not touch the transitive dependencies in the test package. It serves only to validate the packaged artifacts.